core/src/test/java/io/questdb/test/cutlass/text/ParallelCsvFileImporterTest.java

```diff
@@ -701,6 +701,23 @@ public long read(int fd, long buf, long len, long offset) {
         });
     }
 
+    @Test
+    public void testImportFileFailsWhenImportingAfterColumnWasRecreatedNoHeader() throws Exception {
```

Comment:
Does this import fail? Looks like the test name is not in sync with the body.

Reply:
It's leftover. Fixed in 1d392ba9985cad70c9e7dfc59ac035ddfcc8ec0c